db/migrate/20181105212016_duplicate_deleted_at_state_changed_at.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class DuplicateDeletedAtStateChangedAt < ActiveRecord::Migration
+  def up
+    Account.scheduled_for_deletion.where.has{ deleted_at != nil }.update_all('state_changed_at = deleted_at')
```

Comment:
I prefer that to be part of a script not in a migration.
Also the update_all will block the entire table

Reply:
For on-prem though we should ensure the script is run before the app is started.
I think you can do that in the post hook of the deployConfig.

Reply:
We spoke in a call and we decided to do it in a transaction to make things easier but updating one by one and done in a transaction and ensuring that it raises and error and it does rollback if any fails.

Reply:
OK we decide to make it in the migration. It will be easier
/cc @Martouta